falcon/media/json.py

```diff
@@ -94,6 +94,37 @@ class JSONHandler(BaseHandler):
             ),
         )
 
+    You can also override the JSONEncoder used by using a custom Encoder by
```

Comment:
Maybe we could rephrase this slightly to avoid the repetition of _used_ and _using_?

Reply:
Reworded the line.